app/models/organizational_unit.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Superclass of Team and User
+class OrganizationalUnit < Sequel::Model
```

Comment:
Do we want to use the `association_dependencies` plugin to automatically delete the namespace when a OrgUnit is destroyed?